### PR TITLE
PGA: Improving blog post with Vadim's feedback

### PR DESCRIPTION
**Pull request by campoy**

### CODE REVIEW

**Comment by vmarkovtsev** [comment]
@dpordomingo Your feedback on MLonCode category?

**Comment by vmarkovtsev** [comment]
@campoy All good, but I bet that people will ask directly about GDPR - it is very hot atm.

File: content/post/announcing-pga.md [hunk 1]
```diff
@@ -57,6 +57,22 @@ Similarly, the number of lines of code show the size of Public Git Archive.
 
 ![lines of code of datasets](/post/announcing-pga/loc.png)
 
+## Comparing PGA to GHTorrent and the BigQuery GitHub dataset
+
+Many people asked us already how this dataset compares to two related
+projects: GHTorrent and BigQuery.
```
Comment (bzz) [inline content/post/announcing-pga.md]: Would not it be more accurate to call it here and in title as "GitHub public dataset in BigQuery"?

**Comment by dpordomingo** [comment]
Categories are only used for fetching the three first of `technical` and `culture`.
The rest ones are not exposed anywhere, so lgtm

File: content/post/announcing-pga.md (hunk 1 above)
Comment (vmarkovtsev) [reply content/post/announcing-pga.md]: The official name seems to be "GitHub Data"; I am happy with all the three variants, they are impossible to confuse anyway.

**Pull request merged**